### PR TITLE
[trivial] lntest/node: print amounts in same format

### PR DESCRIPTION
**Pull request by halseth**

### CODE REVIEW

File: lntest/node.go [hunk 1]
```diff
@@ -938,7 +938,8 @@ func (hn *HarnessNode) WaitForBalance(expectedBalance int64, confirmed bool) err
 	err := WaitPredicate(doesBalanceMatch, 30*time.Second)
 	if err != nil {
 		return fmt.Errorf("balances not synced after deadline: "+
-			"expected %v, only have %v", expectedBalance, lastBalance)
+			"expected %v, only have %v",
+			btcutil.Amount(expectedBalance), lastBalance)
```
Comment (joostjager) [inline lntest/node.go]: Alternative could be to change the func argument to `btcutil.Amount`.

Comment (joostjager) [reply lntest/node.go]: `fmt.Errorf` call not formatted according to guide lines.

Comment (halseth) [reply lntest/node.go]: > Alternative could be to change the func argument to btcutil.Amount.

Yeah, I would also prefer that. Turned out we are using the proto fields (which are `int64`) directly many places, so the diff became too big to make it worth it I think.

> fmt.Errorf call not formatted according to guide lines.

We usually try to keep logs and errors over as few lines as possible.

Comment (halseth) [reply lntest/node.go]: Actually, the diff didn't become that bad, so I'll do what you initially suggested :)

Comment (joostjager) [reply lntest/node.go]: >We usually try to keep logs and errors over as few lines as possible.

Ah, I see now that that exception indeed landed in the contribution guidelines.

**Pull request merged**